### ISSUE TITLE
[Quest][Zul'Drak] Wooly Justice

### ISSUE THREAD

**Issue by maxipotter** [dalaranwow/dalaran-wow#5373]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Once you ride the Enraged Mammoth every time you use the Trample (the damaging ability), the afected Mam'Toth Disciple trolls lose agro and get reset, making killing them kinda hard. It IS doable since some trolls don't show this bug (but most do) and it's also possible to get some trolls (though not all) to target you quickly after they reset but before they heal up so the damage can stack and kill them.
**Expected behaviour**: Tell us what should happen instead
The trolls should mantain agro and keep attacking the mammoth (player).
**Steps to reproduce the problem**:

1. Climb onto the Mammoth vehicle
2. Attack a Mam'Toth Disciple troll with the Trample
3. Maybe try a couple of times since some trolls don't show this bug

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Here's how it should work: https://www.youtube.com/watch?v=ciAMuTzF-4U&feature=youtu.be

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

Wooly Justice: Quest=12707
Mammoth: npc=28851
Trample: Spell=52603
Troll: npc=28861

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Fix for that needs https://github.com/TrinityCore/TrinityCore/commit/96f90381e3ed696861354ea4ddf4ac0c495031c5
